client/trino-client/src/main/java/io/trino/client/ClientSession.java

```diff
@@ -303,6 +313,7 @@ public static final class Builder
         private Map<String, String> credentials = ImmutableMap.of();
         private String transactionId;
         private Duration clientRequestTimeout;
+        private DataSize targetResultSize = DataSize.of(1, MEGABYTE); // same as DEFAULT_TARGET_RESULT_SIZE
```

Comment:
Nit: adding `in the server` makes it more obvious why we don't use the same constant here.
```suggestion
        private DataSize targetResultSize = DataSize.of(1, MEGABYTE); // same as DEFAULT_TARGET_RESULT_SIZE in the server
```